### DISCUSSION TITLE
How to convert System.Assembly to byte[]

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/runtime#82517] [Q&A]
if I loaded assembly via Assembly.Load and I don't have the bytes anymore, how do I turn that assembly back to bytes

**Comment by am11** [comment]
[`AssemblyBuilder.Save`](https://learn.microsoft.com/en-us/dotnet/api/system.reflection.emit.assemblybuilder.save?view=netframework-4.8.1) is available in .NET Framework, but not in .NET (yet): https://github.com/dotnet/runtime/issues/62956. This API will allow saving `Assembly` objects to file which you can turn into byte array.

**Reply by Xyncgas** [reply]
Thanks. But I'm looking for turning Assembly.CurrentExecuting() Assembly to bytes without saving it to a file, because there is no file system in blazor webassembly
